pymc_experimental/statespace/filters/distributions.py

```diff
@@ -302,71 +364,67 @@ class SequenceMvNormal(Continuous):
     rv_op = KalmanFilterRV
 
     def __new__(cls, *args, **kwargs):
-        support_shape = get_support_shape(
-            support_shape=None,
-            shape=None,
-            dims=kwargs.get("dims", None),
-            observed=None,
-            ndim_supp=2,
-        )
-
-        return super().__new__(cls, *args, support_shape=support_shape, **kwargs)
+        return super().__new__(cls, *args, **kwargs)
 
     @classmethod
-    def dist(cls, mus, covs, logp, support_shape=None, **kwargs):
-        support_shape = get_support_shape(
-            support_shape=None,
-            shape=None,
-            dims=kwargs.get("dims", None),
-            observed=kwargs.get("observed", None),
-            ndim_supp=2,
-        )
-
-        if support_shape is None:
-            support_shape = pt.as_tensor_variable(())
-
-        steps = pm.intX(mus.shape[0])
-
-        return super().dist([mus, covs, logp, steps, support_shape], **kwargs)
+    def dist(cls, mus, covs, logp, **kwargs):
+        return super().dist([mus, covs, logp], **kwargs)
 
     @classmethod
-    def rv_op(cls, mus, covs, logp, steps, support_shape, size=None):
-        if size is not None:
-            batch_size = size
+    def rv_op(cls, mus, covs, logp, size=None):
+
+        # TODO: None of this does anything -- what am I doing wrong?
```

Comment:
Need guidance here. I'm not sure how `support_shape` is meant to be used now that signatures are implemented. It seemed to do nothing so I removed it, but I saw that the timeseries distributions in PyMC core don't do this dimshuffling for scan

Reply:
Usually core shape gives us nsteps in the pymc timeseries but that's only needed because we don't have a parameter changing with time.

If you do here, like mu? you should have all the info you need without an explicit "core_shape" argument. 

Was this related to your question?

Reply:
So I'm safe to just remove this in my case, since I get the time from the inputs and I can safely always assume batches will be added to the left?

Reply:
sounds right